purescript.cabal

```diff
@@ -438,7 +438,7 @@ test-suite tests
   build-depends:
     purescript,
     generic-random >=1.5.0.1 && <1.6,
-    hspec ==2.9.2,
+    hspec >= 2.10.7 && < 3,
```

Comment:
`Test.Hspec` is committed to be stable.  Unless you rely on internal particulars of `hspec-core`, any recent version of Hspec will work.